web/static/js/l4d2.js

```diff
@@ -265,16 +423,286 @@ var app = new Vue({
                     this.config = r
                 },
                 error: () => {
-                    lightyear.notify("网络异常，初始化配置表失败", "danger", 3000);
+                    lightyear.notify("网络异常，初始化配置表失败", "danger", 3000, "", "top", "right");
                 }
             })
-        }
+        },
+        // mod界面函数
+        // 处理选中的文件
+        handleFiles(files) {
```

Comment:
建议删掉注释，用明确的函数名来体现功能，比如
```suggestion
        handleSelectedFiles(files) {
```
或者
```suggestion
        updateSelectedFilePreviews(files) {
```